_chapters/create-containers.md

```diff
@@ -38,8 +38,7 @@ export default App;
 We are doing a few things here:
 
 1. Creating a fixed width container using Bootstrap in `div.container`.
-2. Adding a Navbar inside the container that fits to its container's width using the attribute `fluid`.
```

Comment:
Removing this reference since is not in the code example.

Reply:
Good catch!